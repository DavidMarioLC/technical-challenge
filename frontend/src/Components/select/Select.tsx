import "./select.css"

interface Props {
  defaultValue:string;
  options:Array<string>;
}

const Select = ({defaultValue, options}:Props) => {
  
  
  return (
   <select className="select">
     <option value={defaultValue}>{defaultValue}</option>
     {
       
       options.map((option,index)=>(
        <option key={index} value={option}>{option}</option>
       ))
     }
    
 
   </select>
  )
}

export default Select
