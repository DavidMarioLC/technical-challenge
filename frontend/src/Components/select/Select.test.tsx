
import renderer from 'react-test-renderer';
import Select from './Select';

it('renders Select correctly', () => {
  const tree = renderer.create( <Select setValueSelect={()=>{}}  defaultValue={''}  options={[]}/>).toJSON();
  expect(tree).toMatchSnapshot();
});