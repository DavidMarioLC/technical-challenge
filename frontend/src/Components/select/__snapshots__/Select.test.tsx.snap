// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Select correctly 1`] = `
<select
  className="select"
  onChange={[Function]}
  value=""
/>
`;
